### ISSUE TITLE
SharpVectors throws DomException when loading the specific svg file

### ISSUE THREAD

**Issue by paulushub** [ElinamLLC/SharpVectors#55]
SharpVectors throws DomException when loading the specific svg file

#### Attachments
[aerial-clipart-p-car-top-view.svg](https://www.codeplex.com/Download/AttachmentDownload.ashx?ProjectName=sharpvectors&WorkItemId=2066&FileAttachmentId=1725)

#### This work item was migrated from CodePlex
CodePlex work item ID: '2066'
Vote count: '1'

**Comment by paulushub** [comment]
[popo_lan1986@2017/04/23]
When I tried to convert this svg file, I got a DomException from SharpVectors. 

Inner exception 1
Error message:
    Input string was not in a correct format.
Exception:
    FormatException
Stack trace:
   at System.Number.ParseDouble(String value, NumberStyles options, NumberFormatInfo numfmt)
   at SharpVectors.Dom.Svg.SvgNumber.ParseNumber(String str)

I read the code and found the problem is in WpfSvgPaint.GetDashArray method. We get the value of stroke_dasharray property and parse this value in SvgNumber. In the attached svg file, we can see the value of stroke_dasharray property is "null" and only the three cases null, empty string, "none" are handled. It seems that "null" should be treated like "none".
